jiant/tasks/evaluate/core.py

```diff
@@ -619,7 +619,11 @@ def compute_metrics_from_accumulator(
             null_score_diff_threshold=task.null_score_diff_threshold,
             tokenizer=tokenizer,
         )
-        return Metrics(major=(results["f1"] + results["exact"]) / 2, minor=results,)
+        if task.version_2_with_negative:
+            # Return the score after the best thresholds for answering has been selected
+            return Metrics(major=(results["best_f1"] + results["best_exact"]) / 2, minor=results)
+        else:
+            return Metrics(major=(results["f1"] + results["exact"]) / 2, minor=results)
```

Comment:
Can the naming of `best_f1` just be changed to `f1` for consistency? Or is the distinction in the comment important enough to warrant a different field name?

Reply:
It comes from upstream from HF's SQuAD evaluation code. There's a separate f1/exact field that was previously being incorrectly used.